package.json

```diff
@@ -53,6 +53,7 @@
   "license": "GNU",
   "scripts": {
     "build": "gatsby build",
+    "postbuild": "rm -r ./dist; mv public dist",
```

Comment:
Should we pipe this to `/dev/null` or are we okay with our logs having a bunch of `rm: ./dist: No such file or directory` for each new build?